### PR TITLE
Update CLI docs for 0.70.0 release

### PR DESCRIPTION
**Pull request by eeyun**
Signed-off-by: Ian Henry <ihenry@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/partials/docs/_reference-template-data.html.md.erb [hunk 1]
```diff
@@ -6,21 +6,7 @@ These configuration settings are referenced using the [Handlebars.js](https://gi
 
 ## sys
 
-Describes the details of how this specific Supervisor was started
```
Comment (mwrock) [inline www/source/partials/docs/_reference-template-data.html.md.erb]: seems strange that this block is removed givving us an empty heading

Comment (eeyun) [reply www/source/partials/docs/_reference-template-data.html.md.erb]: That... is very strange. This was generated with the tooling so i wonder if that heading should be removed completely now or something?

Comment (christophermaier) [reply www/source/partials/docs/_reference-template-data.html.md.erb]: Ugh... this has happened for the past few releases... CN was going to look into it, but I guess he wasn't able to before he left.

The issue is that this data is defined in a JSON schema file that is included in several places. Our tooling to automatically generate this documentation from the JSON schema was written back before this was an included file, and thus it does not understand how to deal with included files.

I'll file an issue for this. In the meantime, this whole deletion can just be dropped from this PR (none of this data has changed for this release). @eeyun Could you also make a note of this in the release documentation,  pending the proper solution of this issue?

Comment (christophermaier) [reply www/source/partials/docs/_reference-template-data.html.md.erb]: Issue filed: #5948

**Pull request closed without merging**